panic with oidc authentication

Hello,

running `tfo show` or `tfo help` makes the cli exit with the following error and stack trace:

```
panic: no Auth Provider found for name "oidc"

goroutine 1 [running]:
github.com/isaaguilar/terraform-operator-cli/cmd.newSession()
	/Users/isa/git/terraform-operator-cli/cmd/root_cmd.go:76 +0x235
github.com/spf13/cobra.(*Command).preRun(0x2425ee0)
	/Users/isa/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:886 +0x42
github.com/spf13/cobra.(*Command).execute(0x2425ee0, {0x24699c8, 0x0, 0x0})
	/Users/isa/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:822 +0x1db
github.com/spf13/cobra.(*Command).ExecuteC(0x24259e0)
	/Users/isa/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/isa/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/isaaguilar/terraform-operator-cli/cmd.Execute(...)
	/Users/isa/git/terraform-operator-cli/cmd/root_cmd.go:399
main.main()
	/Users/isa/git/terraform-operator-cli/main.go:10 +0x53
```

This is with tfo version 1.1.2, and authentication via oidc.

--- a/cmd/root_cmd.go
+++ b/cmd/root_cmd.go
@@ -18,6 +18,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/tools/remotecommand"

